youtube_dl/extractor/drtv.py

```diff
@@ -34,17 +47,28 @@ def _real_extract(self, url):
             raise ExtractorError(
                 'Video %s is not available' % video_id, expected=True)
 
-        video_id = self._search_regex(
-            r'data-(?:material-identifier|episode-slug)="([^"]+)"',
-            webpage, 'video id')
+        if ("nyheder" in url) and ("tv/se" not in url):
+            video_id = self._search_regex(
+                r'data-resource="[^>"]+mu/programcard/expanded/([^>"]+)"',
+                webpage, 'video id')
+        else:
+            video_id = self._search_regex(
+                r'data-(?:material-identifier|episode-slug)="([^"]+)"',
+                webpage, 'video id')
 
         programcard = self._download_json(
             'http://www.dr.dk/mu/programcard/expanded/%s' % video_id,
             video_id, 'Downloading video JSON')
         data = programcard['Data'][0]
 
-        title = data['Title']
-        description = data['Description']
+        title = self._html_search_regex(
+            r'property="og:title"[^>]*content="([^">]+)"',
+            webpage, 'title', default=data['Title'])
```

Comment:
It's preferred because some videos (for example the one in the test I added) in their /mu data json file make the title the same as the video ID, rather than the one published on the site, and the description as just a blank string. Whereas every video I've looked at so far the og:title and description have properly matched the video.